src/Microsoft.OData.Edm/ExtensionMethods/EdmTypeSemantics.cs

```diff
@@ -1216,18 +1221,39 @@ public static EdmPrimitiveTypeKind PrimitiveKind(this IEdmTypeReference type)
         /// <param name="potentialBaseType">The potential base type of the type being tested.</param>
         /// <returns>True if and only if the type inherits from the potential base type.</returns>
         public static bool InheritsFrom(this IEdmStructuredType type, IEdmStructuredType potentialBaseType)
+        {            
+            string fullName = type.FullTypeName();

```

Comment:
>fullName [](start = 19, length = 8)

why do you use the "fullName"?

Reply:
Good catch; I think this is leftover code from a previous iteration that can be removed.

---
In reply to: [493000379](https://github.com/OData/odata.net/pull/1888#discussion_r493000379) [](ancestors = 493000379)